### PR TITLE
Footer Menu with Home Button and Profile Button

### LINKED ISSUE #16: Storybook: Footer Menu with Home Button and Profile Button
When I am using the app, I want to be access the Home and Profile at all times, so I can change between those 2 pages.

Create a Storybook Footer Menu with Home Button and Profile Button like we have on the wireframes.

Screenshot 2020-11-15 at 11 14 23

Create the components with the appropriate style and make sure that it works for Desktop and Mobile.

Please check the resources here https://www.figma.com/file/9gN79UmpDPrAZk5jY4V7Ww/Study-Group-Management-System?node-id=0%3A1

### PR DESCRIPTION
**Pull request by nuha90**
# Description

responsive footer menu was created

Fixes #16 

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by tpetrina — APPROVED** [review]
I'll need to check out how it looks, but the code is fine.

File: src/client/components/footer/footer.styles.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  position: absolute;
+  left: 72%;
+  top: 40px;
+}
+.home-paragraph {
+  position: absolute;
+  left: 19%;
+  top: 40px;
+}
+@media screen and (max-width: 600px) {
```
Comment (Pallabi-ghosh) [inline src/client/components/footer/footer.styles.css:48]: `/* Medium devices (landscape tablets, 768px and up) */
@media only screen and (min-width: 768px) {...}`
Nuha, code is looking perfect :)  but I am little bit confused about this Media Query. I mean for mobile and desktop version.

Comment (nuha90) [reply src/client/components/footer/footer.styles.css:48]: it is responsive with mobile check the code in storybook and tell me

Comment (Yaser-Aldammad) [reply src/client/components/footer/footer.styles.css:48]: good job nuha90

Comment (Yaser-Aldammad) [reply src/client/components/footer/footer.styles.css:48]: I have checked the responsive style and changed for min-width and worked for me

**Review by Yaser-Aldammad — APPROVED** [review]
I have checked the responsive style and changed for min-width and worked for me

**Pull request merged**